src/hooks/use-archive-site.ts

```diff
@@ -7,18 +7,25 @@ import { getIpcApi } from '../lib/get-ipc-api';
 import { isWpcomNetworkError } from '../lib/is-wpcom-network-error';
 import { useArchiveErrorMessages } from './use-archive-error-messages';
 import { useAuth } from './use-auth';
-import { SnapshotStatusResponse } from './use-delete-snapshot';
 import { useSiteDetails } from './use-site-details';
+import { useSnapshots } from './use-snapshots';
+
+interface SnapshotStatusResponse {
+	is_deleted: string;
+	domain_name: string;
+	atomic_site_id: string;
+	status: '1' | '2';
+}
```

Comment:
This type is already declared in `src/hooks/use-snapshots.tsx`. Let's import it.

```suggestion
import { SnapshotStatusResponse, useSnapshots } from './use-snapshots';
```